script.js

```diff
@@ -1,4 +1,129 @@
+// Variable declaration
+var gameMode = 'select dice number'
+var numberOfDice = 0;
+var p1Rolls = [];
+var p2Rolls = [];
+var p1combinedNumber = 0;
+var p2combinedNumber = 0;
+
+// Function 1: randomizer between 1-6
+var diceRoll = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+};
+
+// Function 2: rolls dice X number of times
+var rollsInArray = function(diceCount){
+  var counter = 0;
+  var thisArray = [];
+  while (counter < diceCount){
+    thisArray.push(diceRoll());
+    counter = counter + 1;
+  }
+  return thisArray;
+}
+
+// Function 3: print the roll result
+var printRollResult = function(playerArray){
+  var msgResult = '';
+  var lengthArray = playerArray.length;
+  var firstLine = 'You rolled:';
+  var secondLine = '<br> ----dice:'
+  var thirdLine = '<br> position:'
+  var fourthLine = '<br> Enter the order of the positions of the numbers you want'
+  var counter = 0;
+  while (counter < lengthArray){
+    secondLine = secondLine + ' ' + playerArray[counter] ;
+    thirdLine = thirdLine + ' ' +counter;
+    counter = counter + 1;
+  }
+  msgResult = firstLine + secondLine + thirdLine + fourthLine
+  return msgResult; 
+}
+
+// Function 4: Creates number from number input
+var numberOrdered = function (userInputOrder, userDice){
```

Comment:
This function name is confusing- would recommend a clearer name!

Reply:
`userInputOrder` is relatively clear, `userDice` isn't super clear

Reply:
Would recommend trying JSDoc comment format for clearer function-level commenting https://jsdoc.app/about-getting-started.html